x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/rollup/job/TermsGroupConfig.java

```diff
@@ -94,14 +103,6 @@ private TermsGroupConfig(String[] fields) {
         return map;
     }
 
-    public Map<String, Object> getMetadata() {
-        return Collections.emptyMap();
```

Comment:
Just a note: this is a no-op for Terms, but the other groups (date, histo) use it to store the interval.  So we'll need to preserve that functionality somehow/somewhere for the other groups

Reply:
I agree, we'll keep this for the other groups.